KEEP/initramfs.sh

```diff
@@ -66,4 +67,3 @@ umount /sys /proc
 [ -d /boot ] && umount /boot
 mount --move /dev /root/dev
 exec switch_root /root "${init}"
-rescue "switch_root failed with $?"
```

Comment:
is it safe to remove this ? i remember that i was being dropped here in the rescue shell when my cryptsetup was not being done due to the previous bug